modules/_node-scripts/format/index.mjs

```diff
@@ -3,17 +3,21 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import path from 'path';
+
 import preflight from '../preflight/index.mjs';
+import {getRootDir} from '../util/constants.mjs';
 import format from './format.mjs';
 
 export default async function main() {
 	const fix = process.argv[3] !== '--check';
+	const rootDir = await getRootDir();
 
-	if (!fix) {
-		console.log('Running preflight...');
+	if (path.resolve(process.cwd()) === rootDir) {
```

Comment:
Why do you wrap cwd in resolve?

Reply:
To make it an absolute path. Just in case... :grimacing: